tests/Api/StatsTest.php

```diff
@@ -29,14 +29,15 @@ public function testTotal()
         $api->total('domain', $data);
     }
 
-    /**
-     * expectedException \Mailgun\Exception\InvalidArgumentException.
-     */
-    //public function testTotalInvalidArgument()
-    //{
-    //    $api = $this->getApiMock();
-    //    $api->total('');
-    //}
+    // /**
+    //  * @expectedException \Mailgun\Exception\InvalidArgumentException
+    //  */
+    // public function testTotalInvalidArgument()
+    // {
+    //     $api = $this->getApiMock();
+    //
+    //     $api->total('');
+    // }
```

Comment:
Sorry. My bad. I made that mistake.  That is a test that should be used when asset 1.3 is released.